list.c

```diff
@@ -25,11 +25,23 @@
  * 1 tab == 4 spaces!
  */
 
-
+/* Standard includes. */
 #include <stdlib.h>
+
+/* Defining MPU_WRAPPERS_INCLUDED_FROM_API_FILE prevents task.h from redefining
+all the API functions to use the MPU wrappers.  That should only be done when
+task.h is included from an application file. */
+#define MPU_WRAPPERS_INCLUDED_FROM_API_FILE
+

```

Comment:
Why is this required in list.c?

Reply:
Otherwise PRIVILEGED_FUNCTION does not get defined and as a result list functions get placed in unprivileged region.

https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/master/include/mpu_wrappers.h#L165